src/pokemon_pb.erl

```diff
@@ -107,13 +107,13 @@ int_to_enum(_,Val) ->
     Val.
 
 %% DECODE
-decode_pikachu(Bytes) when is_binary(Bytes) ->
```

Comment:
This seems like an unnecessary change. @javajolt @alexmoore @tburghart - care to weigh in?